x/mongo/driver/operation.go

```diff
@@ -471,8 +474,9 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 			}
 		}
 
-		// Calculate value of 'maxTimeMS' field to potentially append to the wire message based on the current
-		// context's deadline and the 90th percentile RTT if the ctx is a Timeout Context.
+		// Calculate value of 'maxTimeMS' field to potentially append to the wire message based on the
+		// current context's deadline and the 90th percentile RTT if the ctx is a Timeout Context. If
+		// context is not a Timeout context, use the passed in MaxTimeMS if set.
```

Comment:
`maxTimeMS` appension is now done here in `Operation.Execute` instead of in respective operations' `command` functions in order to unify the logic of `maxTimeMS` appension based on whether `Timeout` has been set on the `Client`.